index.html

```diff
@@ -77,14 +74,18 @@ <h4>January 2021, Bristol UK</h4>
 
 
 	<div class="aboutme" id="aboutme">
-		<div class="container aboutmebackground">
-			
+		<div class="container aboutmebackground">		
 			<div class="aboutmebox">	
-				<img src="jennyaboutme.jpg" alt="Jenny Allton-Twist looking happy"/>
+				<img class="aboutmeimg" src="jennyaboutme.jpg" alt="Jenny Allton-Twist looking happy"/>
 			</div>
 			<div class="aboutmebox">
-				<h2>About Me</h2>
-				<p>Some stuff about me, a paragraph in which I talk humorously and succintly about my many achievements and accolades, as well as the interesting and altruistic activities I undertake in my spare time.</p>
+				<h2>ABOUT ME</h2>
+				<p>I’m a full-stack software developer with an interest in back-end development and machine learning. I graduated from<a href="https://mayden.academy/"> Mayden Academy</a> in December 2020.</p>
+				<p>Prior to my formal training I have almost a decade’s experience in project management and product design in a superyacht interior design company. My strengths are in my problem-solving and communication skills, both written and verbal.</p>
+				<p>I love running, reading trashy urban fiction, and learning languages that are spoken by people as well as languages used by computers. Oh, and I really like burritos.</p>
+				<p></p>
```

Comment:
Is this extra paragraph tag intentional?

Reply:
Yup, it was to give space before the final p.